### PR TITLE
Draft reflection and update README

### LINKED ISSUE #31: (milestone 2) 6. Improve README
Include motivation behind your project and clearly explain what problem you are solving and why it is important.

You do not have to include detailed usage instructions, just high level what they can do in your dashboard and and the deployed link.

[This is a good example](https://github.com/KirstieJane/STEMMRoleModels)

### LINKED ISSUE #33: (milestone 2) 5. Reflection
In this section, your group should document on what you have implemented in your dashboard so far and explain what is not yet implemented. It is important that you include what you know is not working in your dashboard, so that your TAs can distinguish between features in development and bugs.

Reflect on what you think your dashboard does well what its limitations are, and what are good future improvements and additions. This section should not be more than 500 words and the reflection-milestone2.md document should live in your GitHub.com repo in the doc folder.

### PR DESCRIPTION
**Pull request by celestezhao**

### CODE REVIEW

**Comment by JohnShiuMK** [comment]
I have tested the installation guide in the README, had some warnings, have already updated the environment.yml and requirements.txt.
They work fine now

File: README.md [hunk 1]
```diff
@@ -1,23 +1,58 @@
...
 
-## Purpose
+[![GitHub issues](https://img.shields.io/github/issues/UBC-MDS/DSCI-532_2024_19_food-price-tracker.svg)](https://github.com/UBC-MDS/DSCI-532_2024_19_food-price-tracker/issues)
+[![GitHub last commit](https://img.shields.io/github/last-commit/UBC-MDS/DSCI-532_2024_19_food-price-tracker.svg)](https://github.com/UBC-MDS/DSCI-532_2024_19_food-price-tracker/commits/main)
+[![GitHub release](https://img.shields.io/github/release/UBC-MDS/DSCI-532_2024_19_food-price-tracker.svg)](https://github.com/UBC-MDS/DSCI-532_2024_19_food-price-tracker/releases)
 
-Access to reliable food pricing information is crucial for informed decision-making in policymaking and business strategy.
-However, existing platforms lack comprehensive global trends with regional detail, posing challenges for non-technical users.
-The project aims to provide an intuitive, interactive data visualization platform to empower stakeholders for food price insights and enhance cross-sector collaboration to address worldwide food-related challenges. 
+![logo](https://raw.githubusercontent.com/UBC-MDS/DSCI-532_2024_19_food-price-tracker/img/logo.png)
```
Comment (srfrew) [inline README.md]: Should be updated to a relative path within the repository

File: README.md [hunk 2]
```diff
@@ -1,23 +1,58 @@
...
+
+## Usage - For Developers
+
+### Installation For Local Developing
+
+1. Clone this repository to your computer.
 
 ```bash
- git clone git@github.com:UBC-MDS/DSCI-532_2024_19_food-price-tracker.git
+ git clone https://github.com/UBC-MDS/DSCI-532_2024_19_food-price-tracker
```
Comment (srfrew) [inline README.md]: Recommend SSH based installation over HTTPS

File: README.md [hunk 3]
```diff
@@ -32,21 +67,13 @@ The project aims to provide an intuitive, interactive data visualization platfor
...
-## Contributing
+### Contributing
 
-Interested in contributing? Check out the contributing guidelines. Please note that this project is released with a Code of Conduct. By contributing to this project, you agree to abide by its terms.
+Interested in contributing? Check out the [contributing guidelines](CONTRIBUTING.md). Please note that this project is released with a [Code of Conduct](CODE_OF_CONDUCT.md). By contributing to this project, you agree to abide by its terms.
 
 ## License
 
 `food_price_tracker` was created by Celeste Zhao, John Shiu, Simon Frew, Tony Shum. It is licensed under the terms of the MIT license and the Attribution 4.0 International (CC BY 4.0 LEGAL CODE).
-
```
Comment (srfrew) [inline README.md:49]: I believe these credits and attributions are still required

File: reports/m2-reflection.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Sidebar: Users can select the country, time range, specific commodities, and markets they wish to view.
+- `Country` dropdown: Allows users to choose the country they are interested in.
+- `Date` selection: Enables selection of the time range for data viewing.
+- `Commodities` dropdown: Users can select the range of commodities to be examined.
+- `Markets` dropdown: Allows for the selection of specific markets.
+- View toggle (future update): Switches to a map perspective "Geo view".
+- `Manual Trigger`: Runs the dashboard based on selected conditions.
+
+2. Data Display Section: Displays food price information based on selected conditions.
```
Comment (srfrew) [inline reports/m2-reflection.md]: Phrasing of the "line chart vs box chart" can be changed to improve readability

File: reports/m2-reflection.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- Food price index line chart: Displays the Food Price Index values over time across different markets based on selected conditions.
+- Individual commodity tracking charts: Below a separator, this shows the average price, Month-over-Month change rate, and Year-over-Year change rate of USD prices for each commodity within the selected time and market range.
+- Individual commodity line charts: Displays the USD prices over time for each commodity across different markets based on selected conditions.
+
+## 2. Features in Proposal Yet To Be Done
+
+In our Milestone 1 Proposal, we planed to implement a toggle to switch from the current view to a "Geo view". In the Geo view, users would see a country map with the latest prices from various markets for specific commodities on a given date, enabling regional price comparisons. However, due to the substantial workload and limited time, this feature has not yet been implemented.
+
+## 3. Features Not Working Well and Limitations of Current Dashboard
```
Comment (srfrew) [inline reports/m2-reflection.md:27]: Note to update this section for current and resolved issues

**Review by srfrew — CHANGES_REQUESTED** [review]
Overall very well done draft! I noted some changes for clarity and will edit accordingly

File: README.md [hunk 6]
```diff
@@ -35,18 +71,14 @@ The project aims to provide an intuitive, interactive data visualization platfor
...
+Interested in contributing? Check out the [contributing guidelines](CONTRIBUTING.md). Please note that this project is released with a [Code of Conduct](CODE_OF_CONDUCT.md). By contributing to this project, you agree to abide by its terms.
 
 ## License
 
 `food_price_tracker` was created by Celeste Zhao, John Shiu, Simon Frew, Tony Shum. It is licensed under the terms of the MIT license and the Attribution 4.0 International (CC BY 4.0 LEGAL CODE).
 
 ## Credits
 
-`food_price_tracker` was created with [`cookiecutter`](https://cookiecutter.readthedocs.io/en/latest/) and the `py-pkgs-cookiecutter` [template](https://github.com/py-pkgs/py-pkgs-cookiecutter).
+`food_price_tracker` was created with [`cookiecutter`](https://cookiecutter.readthedocs.io/en/latest/) and the `py-pkgs-cookiecutter` [template](https://github.com/py-pkgs/py-pkgs-cookiecutter). Logo created by DALL.E.
```
Comment (JohnShiuMK) [inline README.md]: do we also need to credit the data source?

Comment (srfrew) [reply README.md]: Yes, I thought it was previously linked bit it seems that it is not. I'll update it now

**Comment by srfrew** [comment]
This is ready for review and merge @JohnShiuMK

**Review by srfrew — APPROVED** [review]

**Comment by JohnShiuMK** [comment]
look really good!
deeply appreciate for @celestezhao and @srfrew effort!

**Pull request merged**